mappings/net/minecraft/client/gui/screen/world/EditGameRulesScreen.mapping

```diff
@@ -0,0 +1,54 @@
+CLASS net/minecraft/class_5235 net/minecraft/client/gui/screen/world/EditGameRulesScreen
+	FIELD field_24293 ruleSaveConsumer Ljava/util/function/Consumer;
+	FIELD field_24294 ruleListWidget Lnet/minecraft/class_5235$class_5241;
+	FIELD field_24295 invalidRuleWidgets Ljava/util/Set;
+	FIELD field_24296 doneButton Lnet/minecraft/class_4185;
+	FIELD field_24297 tooltipDescription Ljava/util/List;
```

Comment:
Hmm, isn't a tooltip already a description? mind elaborate on this field?

Reply:
Hmm yeah Its the tooltip